conda/cli/main_env_update.py

```diff
@@ -109,6 +109,8 @@ def execute(args: Namespace, parser: ArgumentParser) -> int:
     )
     spec = spec_hook.environment_spec(args.file)
     env = spec.env
+    # reset the context to pick up config from the environment spec
+    context.__init__(argparse_args=args, env_spec_config=env.config)
```

Comment:
I think since the context needs to be re-initialized after the env spec pulgins have run, the env spec plugins should not be allowed